run_scripts/start_bag.il

```diff
@@ -1441,15 +1458,6 @@ procedure( create_layout( lib_name view_name via_tech layout_f "ttt" )
             printf("Creating %s__%s (%s)\n" lib_name cell_name view_name)
             create_layout_helper(cv tech_file inst_list rect_list via_list pin_list path_list
                                  blockage_list boundary_list polygon_list)
-            
-            if( exists( inst inst_list inst->params != nil ) ; Check whether cellview contains any Pcell instances
-                if( getd('CCSinvokeCdfCallbacks) != nil then ; Found function handle for callbacks, run callbacks 
-                    printf("Running callbacks for cellview: %s\n" cell_name)
-                    (CCSinvokeCdfCallbacks cv)
```

Comment:
This way of invoking callbacks seems to update layouts perfectly fine. However, it takes an excessive amount of time to create layouts.